<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionTrackerDialog</class>
 <widget class="QDialog" name="MotionTrackerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="2" column="1">
    <widget class="QLabel" name="diffThreshLabel">
     <property name="text">
      <string>Diff Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="blurLabel">
     <property name="text">
      <string>Blur, px:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="segThreshLabel">
     <property name="text">
      <string>Segment Threshold, ms:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSlider" name="rectMinSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="durationSpinBox"/>
   </item>
   <item row="7" column="2">
    <widget class="QSlider" name="rectMaxSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSlider" name="diffThreshSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="blurSpinBox"/>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="diffThreshSpinBox"/>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="segThreshSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="blurSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSlider" name="deltaMaxSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="deltaMinLabel">
     <property name="text">
      <string>Motion Gradient Delta, min:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSlider" name="durationSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="deltaMaxLabel">
     <property name="text">
      <string>Motion Gradient Delta, max:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="rectMinLabel">
     <property name="text">
      <string>Drawable Rect Area, min:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="rectMaxLabel">
     <property name="text">
      <string>Drawable Rect, Area, max:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Motion History Duration, ms:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSlider" name="deltaMinSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QDoubleSpinBox" name="segThreshSpinBox"/>
   </item>
   <item row="4" column="3">
    <widget class="QDoubleSpinBox" name="deltaMinSpinBox"/>
   </item>
   <item row="5" column="3">
    <widget class="QDoubleSpinBox" name="deltaMaxSpinBox"/>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="rectMinSpinBox"/>
   </item>
   <item row="7" column="3">
    <widget class="QSpinBox" name="rectMaxSpinBox"/>
   </item>
   <item row="9" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QCheckBox" name="orientation">
     <property name="text">
      <string>Draw Orientation Circle</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QToolButton" name="wtfOrientation">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QToolButton" name="wtfRectMax">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QToolButton" name="wtfRectMin">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QToolButton" name="wtfDeltaMax">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QToolButton" name="wtfDeltaMin">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="wtfSegThresh">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="wtfDiffThresh">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="wtfBlur">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="wtfDuration">
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MotionTrackerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MotionTrackerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
